python/runtime/feature/field_desc.py

```diff
@@ -26,6 +26,7 @@ class DataType(object):
     INT = 0
     FLOAT = 1
     STRING = 2
+    INT64 = 3
```

Comment:
We can keep only one of `INT` or `INT64`?

Reply:
Need to map dtype to TensorFlow data types finally, int64 is needed here.